doc/045_working_with_repos.rst

```diff
@@ -285,3 +285,16 @@ integer:
 .. code-block:: console
 
     $ restic -r /srv/restic-repo check --read-data-subset=10%
+
+Use ``--read-data-subset=SB`` to check a randomly choosen subset of the
+repository pack files. It takes one parameter, ``SB``, the required bytes (KB/MB/GB/TB) of pack
+files to check. Behind the scenes, the specified bytes will be converted to percentage 
+of the total repository size. The behaviour of the check command following this conversion will be the same as the percentage option above.
```

Comment:
This line seems too wide. It's 139 characters. The rest of the file seems to try to stay under 80.